src/js/angular/core/services.js

```diff
@@ -290,12 +290,14 @@ function ClassInstanceDetailsService($http) {
 
 function AuthTokenService() {
     const authStorageName = 'com.ontotext.graphdb.auth';
+    const OPENID_CONFIG = {};
```

Comment:
All open ID configurations will be in memory. I think this will break some logout flows when the user logged out from a second tab.

Reply:
They were in momory before also. They are fetched on security init.
I just moved them so that they are accessable from other places (`authentication.interceptor`)

Reply:
There was a problem and I have done a fix